### ISSUE TITLE
But when using "2>&1" in exec command

### ISSUE THREAD

**Issue by digirave** [OOPS-ORG-PHP/mod_execdir#15]
Problem:
When adding "2>&1" to exec argument, it causes commands to fail

How to reproduce problem:
execdir.ini setting:
extension = execdir.so
exec_dir = /etc/run

php code:
$a = "/etc/run/pngquant '--force' '--ext=.png' '--' '/testimage/a.png' 2>&1";
print exec($a);

error:
fails with
sh: /etc/run/: ambiguous redirect

workaround:
currently must delete "2>&1"
$a = "/etc/run/pngquant '--force' '--ext=.png' '--' '/testimage/a.png'";
print exec($a);
will work

**Comment by Joungkyun** [comment]
Most of all redirection seems to be a problem. First, I marked it with a bug.

However, we need thinking to decide whether to allow this policy for a few days.

If you are wondering how the command changes with mod_execdir, you can prefix the command with the prefix "DEBUG:".

```php
<?php
$a = "DEBUG:pngquant '--force' '--ext=.png' '--' '/testimage/a.png' 2>&1";
echo system($a). "\n";
?>
```

**Comment by Joungkyun** [comment]
Right. This is definitely a bug. It will be fixed within a few days.

**Comment by Joungkyun** [comment]
fixed with 5879fdfc4eb47d5d753af9b8eea58519110918ab
